web/src/components/Plugin/data.tsx

```diff
@@ -0,0 +1,169 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
```

Comment:
@nic-chen @membphis Do we support fetching plugin list with the category?

Reply:
![image](https://user-images.githubusercontent.com/2106987/100491073-eba24f80-315b-11eb-8768-855bceff7289.png)

I found this API: https://github.com/apache/apisix/blob/master/doc/admin-api.md#plugin-metadata

It seems that the manager-api doesn't support fetching Metadata 🤔 not urgent.

Reply:
manager-api doesn't support fetching Metadata yet.
but why we need metadata here ?

Reply:
not sure, I just want to fetch plugin list with category.